stdlib/experimental/chain_test.go

```diff
@@ -0,0 +1,110 @@
+package experimental_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/influxdata/flux/dependencies/dependenciestest"
+	"github.com/influxdata/flux/lang/execdeps"
+	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/stdlib/experimental"
+	"github.com/influxdata/flux/values"
+)
+
+var table1 = `
+import "csv"
+
+data = "#datatype,string,long,dateTime:RFC3339,double,string,string
+#group,false,false,false,false,true,true
+#default,_result,,,,,
+,result,table,_time,_value,_measurement,user
+,,0,2018-05-22T19:53:26Z,0,CPU,user1
+,,0,2018-05-22T19:53:36Z,1,CPU,user1
+,,1,2018-05-22T19:53:26Z,4,CPU,user2
+,,1,2018-05-22T19:53:36Z,20,CPU,user2
+,,1,2018-05-22T19:53:46Z,7,CPU,user2
+,,2,2018-05-22T19:53:26Z,1,RAM,user1
+"
+
+inj = csv.from(csv: data)
```

Comment:
Does `inj` refer to "injection"? I noticed this in the table find tests (which these are based off of) and took it mean that.

Reply:
I have no clue. I just took it straight from the `tableFind` tests. :)